.github/workflows/build-docs.yaml

```diff
@@ -21,12 +27,10 @@ concurrency:
   cancel-in-progress: true
 
 permissions:
-  contents: read
+  contents: write # for peaceiris/actions-gh-pages to push pages branch
```

Comment:
Why wasn't the job specific override better?

Reply:
Didn't work, the parent had preference